test/src/unit-capi-sparse_heter.cc

```diff
@@ -723,21 +728,8 @@ void SparseHeterFx::check_read_sparse_array_int64_float(
 TEST_CASE_METHOD(
     SparseHeterFx,
     "C API: Test sparse array with heterogeneous domains (float, int64)",
-    "[capi][sparse][heter][float-int64]") {
-  SECTION("- No serialization") {
-    serialize_ = false;
-  }
-  SECTION("- Serialization") {
-#ifdef TILEDB_SERIALIZATION
-    serialize_ = true;
-    refactored_query_v2_ = GENERATE(true, false);
-#endif
-  }
-
-  SupportedFsLocal local_fs;
-  std::string array_name =
-      local_fs.file_prefix() + local_fs.temp_dir() + "sparse_array_heter";
-  create_temp_dir(local_fs.file_prefix() + local_fs.temp_dir());
+    "[capi][sparse][heter][float-int64][non-rest]") {
```

Comment:
It's that I have verified that this test cannot be ported to REST-CI because it's using functionality that's not supported, e.g. encyption, consolidation etc. My thought is that in this way we can see clearly which tests we didn't work on porting yet, and which can't be ported ([non-rest] tagged ones). Let me know if you don't like it.